src/unitxt/splitters.py

```diff
@@ -183,6 +183,10 @@ def process(self, instance: Dict[str, object], multi_stream: MultiStream) -> Dic
             raise Exception(f"Unable to fetch instances from '{self.source_stream}' to '{self.target_field}'")
 
 
+class CreateDemoSplit(SpreadSplit):
+    pass
```

Comment:
what is this part for?

Reply:
To give a meaningful name to that part of the pipeline in debug mode.